client/src/components/ProfileSettings/ManageBooking/ManageBooking.tsx

```diff
@@ -1,19 +1,40 @@
-import { Grid, Container } from '@material-ui/core';
+import { useState, useEffect } from 'react';
+import { Grid, Container, Input } from '@material-ui/core';
 import BookingList from './BookingList/BookingList';
 import CalendarDisplay from './Calendar/CalendarDisplay';
 import useStyles from './useStyles';
+import axios from 'axios';
+import { useAuth } from '../../../context/useAuthContext';
 
 const ManageBooking = (): JSX.Element => {
+  const { loggedInUser } = useAuth();
+  const [incomingData, setIncomingData] = useState({});
   const classes = useStyles();
+
+  useEffect(() => {
+    const handleIncomingBooking = async () => {
+      try {
+        setIncomingData(await axios.post('http://localhost:3001/profile/manage-booking', JSON.stringify(loggedInUser)));
```

Comment:
Same as comment above - define this API call in a function and relocate to the helpers folder